lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -26,8 +26,7 @@ updateScriptTags = (activeDocument, newScripts, callback) ->
   )
 
 extractTrackedAssets = (doc) ->
-  for node in doc.head.children when node.dataset.turbolinksTrack?
-    node
+  [].slice.call(doc.querySelectorAll('[data-turbolinks-track]'))
```

Comment:
It outputs considerable more legible javascript than a coffeescript array comprehension.